metadata-service/restli-client/src/main/java/com/linkedin/entity/client/JavaEntityClient.java

```diff
@@ -349,6 +349,15 @@ public VersionedAspect getAspect(@Nonnull String urn, @Nonnull String aspect, @N
         return _entityService.getVersionedAspect(Urn.createFromString(urn), aspect, version);
     }
 
+    @Override
+    public EntityResponse getVersionedEntity(@Nonnull final String entityName,
```

Comment:
What does it mean here to get a versioned entity? Should this method read:

getVersionedAspect?

Reply:
Specifically it means that it's an EntityResponse. Under that interface it is really just a single aspect, yes, but wanted to avoid reusing "getVersionedAspect" as that is a legacy endpoint on the Aspect side which uses the old "VersionedAspect" union type that manually specifies every possible aspect (except any new ones and does not cover any of the entity-registry use cases).